nox/sessions.py

```diff
@@ -554,7 +554,9 @@ def conda_install(
 
         You can specify a conda channel using `channel=`; a falsey value will
         not change the current channels. You can specify a list of channels if
-        needed.
+        needed. It is highly recommended to specify this; micromamba does not
+        set default channels, and default channels vary for conda. Note that
+        "defaults" is also not permissivly licenced like "conda-forge" is.
```

Comment:
@henryiii This sounds ominous, Anaconda's `defaults` channel is commercially licensed and is free to use for educational users and companies with fewer than 200 employees.